x-pack/plugins/case/common/api/cases/comment.ts

```diff
@@ -18,24 +33,62 @@ export const CommentAttributesBasicRt = rt.type({
   updated_by: rt.union([UserRT, rt.null]),
 });
 
+export const GeneratedAlertRequestTypeField = 'generated_alert_request';
+
+export enum CommentType {
+  user = 'user',
+  alert = 'alert',
+  generatedAlert = 'generated_alert',
```

Comment:
I am a bit confused with `generated_alert_request` and `generated_alert`. Shouldn't `CommentType.generatedAlert === generated_alert_request` ?

Reply:
Christos and I talked about this and I refactored it so we don't need `generated_alert_request` 👍